deprecated/pipelines/cemba/cemba_methylcseq/example_inputs/README.md

```diff
@@ -0,0 +1,9 @@
+## Announcement: CEMBA file paths are deprecated 11/05/2024
```

Comment:
this looks good! but maybe we should add this information to the existing README here? deprecated/pipelines/cemba/cemba_methylcseq/README.md

Reply:
That makes sense. Ill do that.